### DISCUSSION TITLE
Revisão de tradução: WebAIM - Accessible Javascript

### DISCUSSION THREAD

**Discussion by fititnt** [frontendbr/forum#315] [Discussão]
Relacionado a webiwg/acessibilidade-web-pt#22

---

**Colegas eu vou traduzir o [Accessible Javascript](http://webaim.org/techniques/javascript/) da WebAIM para português até semana que vem. Algum colega aqui tem interesse em ser revisor da tradução?**

O link do original que será traduzido mas a [WebIWG](https://github.com/webiwg) aceita ajuda externa pra pra etapa de revisão é http://webaim.org/techniques/javascript/

---

Como referência de traduções anteriores (essas já estão sendo revisadas por outro colega):
- [Como criar formulários acessíveis](http://acessibilidade.pt.webiwg.org/webaim/tecnicas/formularios/), tópico https://github.com/webiwg/acessibilidade-web-pt/issues/12
- [CSS em ação: conteúdo invisível apenas para usuários de leitores de tela](http://acessibilidade.pt.webiwg.org/webaim/tecnicas/css/conteudo-invisivel-apenas-para-leitores-de-tela/), tópico https://github.com/webiwg/acessibilidade-web-pt/issues/16

**Comment by suissa** [comment]
EU!

**Comment by robsmayer** [comment]
Olá

**Comment by robsmayer** [comment]
@suissa acabei de falar com o @fititnt e eu vou assumir a tradução desse texto. Se quiser pode acompanhar o andamento lá em https://github.com/webiwg/acessibilidade-web-pt/pull/25.

Assim que eu terminar ela, você pode assumir a revisão da tradução.

**Comment by suissa** [comment]
Irei revisar a traduçao fds ;)
